### PR TITLE
Document timeout value unit.

### LINKED ISSUE #769: NH-3922 - The various timeout methods should indicate time unit
**Oskar Berggren** created an [issue](https://nhibernate.jira.com/browse/NH-3922) — <time datetime="2016-11-20T14:59:15.524Z" title="2016-11-20T14:59:15.524Z">20th November 2016, 14:59:15</time>:
> IQuery.SetTimeout(timeout)
> LinqExtensionMethods.Timeout(this queryable, timeout)
> possibly more?
> 
> The "timeout" is just an integer parameter. Nor are there code comments to indicate the time unit of the value. We should add comments and rename the variable to timeoutSeconds (I think). Possibly provide a TimeSpan-based overload.
> 
---
**Alexander Zaytsev** added a comment — <time datetime="2017-09-14T01:57:24.714Z" title="2017-09-14T01:57:24.714Z">14th September 2017, 1:57:24</time>:
> Move minor and trivial improvements to 5.1
>

### PR DESCRIPTION
**Pull request by fredericDelaporte**
- Fixes #769 ( NH-3922 )

### CODE REVIEW

**Comment by hazzik** [comment]
What about "Possibly provide a TimeSpan-based overload." part?

**Comment by fredericDelaporte** [comment]
I am not convinced by this part. `DbCommand` takes seconds and does not allow to specify fractional seconds. So a timespan in milliseconds will have to be round or cut (with the risk of transforming a short millisecond timeout into an infinite one, `0`). And specifying timeout of more than one minute for a SQL query should be quite unusual and not encouraged, so having to define such a timeout in seconds only is not very inconvenient in my mind.

**Comment by hazzik** [comment]
Ok

File: src/NHibernate/Async/Driver/IResultSetsCommand.cs [hunk 1]
```diff
@@ -17,6 +17,12 @@ namespace NHibernate.Driver
 	using System.Threading;
 	public partial interface IResultSetsCommand
 	{
+		/// <summary>
+		/// Get a data reader for this multiple result sets command.
+		/// </summary>
+		/// <param name="commandTimeout">The timeout in seconds for the underlying ADO.NET query.</param>
+		/// <param name="cancellationToken">A cancellation token that can be used to cancel the work</param>
+		/// <returns>A data reader.</returns>
```
Comment (hazzik) [inline src/NHibernate/Async/Driver/IResultSetsCommand.cs]: Is this regenerated?

Comment (fredericDelaporte) [reply src/NHibernate/Async/Driver/IResultSetsCommand.cs]: Yes.

**Pull request merged**